src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.service.imp.CsvFileReaderService;
+import core.basesyntax.service.imp.CsvFileWriterService;
+import java.util.List;
+
+public class Main {
+    private static final String READ_FILE_NAME = "source.csv";
+    private static final String WRITE_FILE_NAME = "report.csv";
+
+    public static void main(String[] args) {
```

Comment:
in main method you should create every single service separately and call their method sequentially. For example:
```
ReaderService readerService = new ReaderServiceImpl();
List<String> rows = readerService.readFromFile(READ_FILE_NAME);
....
String report = reportService.createReport();
writerService.writeToFile(report, WRITE_FILE_NAME);
```